libs/platform/src/lib/components/thumbnail/thumbnail.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Input, ViewEncapsulation, Output, EventEmitter } from '@angular/core';
+
+export class Media {
+    thumbnailUrl: string;
+    mediaType: string;
+    mediaUrl: string;
+    alt: string;
+    label: string;
+}
```

Comment:
I think you should use `type` instead of `class` unless you have a reason for making it a `class`.